assets/queries/cloudFormation/BucketPolicyEnabled/test/negative.yaml

```diff
@@ -0,0 +1,66 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: A sample template
+Resources:
+  mybucket:
+    Type: "AWS::S3::Bucket"
+    DeletionPolicy: Retain
+    Properties:
+      ReplicationConfiguration:
+        Role:
+          "Fn::GetAtt":
+            - WorkItemBucketBackupRole
+            - Arn
+        Rules:
+          - Destination:
+              Bucket:
+                "Fn::Join":
+                  - ""
+                  - - "arn:aws:s3:::"
+                    - "Fn::Join":
+                        - "-"
+                        - - Ref: "AWS::Region"
+                          - Ref: "AWS::StackName"
+                          - replicationbucket
+              StorageClass: STANDARD
+            Id: Backup
+            Prefix: ""
+            Status: Enabled
+      VersioningConfiguration:
+        Status: Enabled
+      LoggingConfiguration:
+        DestinationBucketName: !Ref LoggingBucket
+        LogFilePrefix: loga/
+  WorkItemBucketBackupRole:
+    Type: "AWS::IAM::Role"
+    Properties:
+      AssumeRolePolicyDocument:
+        Statement:
+          - Action:
+              - "sts:AssumeRole"
+            Effect: Allow
+            Principal:
+              Service:
+                - s3.amazonaws.com
+  SampleBucketPolicy:
+    Type: 'AWS::S3::BucketPolicy'
+    Properties:
+      Bucket:
+        Ref: mybucket
+      PolicyDocument:
+        Statement:
+          - Action:
+              - 's3:GetObject'
+            Effect: Allow
+            Resource:
+              'Fn::Join':
```

Comment:
Linter is complaining about this, Try removing the surrounding quotes: `Fn::Join`